### PR TITLE
Add floor selector component

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by sbrunner** [comment]
You add it to the desktop alt, but is there a way to test it on the gh-pages?

**Comment by arnaud-morvan** [comment]
You should already view on it on gh-pages, but it will have no effect.
I can open a pull request on demo_geomapfish 2.4 with new mapserver layer, and we might add it in admin interface.

**Comment by arnaud-morvan** [comment]
@sbrunner @fredj 
It is now available on gh pages : https://camptocamp.github.io/ngeo/floorselector/examples/contribs/gmf/apps/desktop_alt
I get a CORS error on the themes request, so I cannot see my new layer configured with the floor slider component.
You can use the up and down buttons, click on the floors or use the mouse wheel.
Note that we might debounce between mousewheel and changing layer parameters.

**Comment by arnaud-morvan** [comment]
Completely works now : https://localhost:3000/contribs/gmf/apps/desktop_alt.html?dim_FLOOR=6&lang=fr&baselayer_ref=blank&tree_groups=floor_slider&map_x=734830&map_y=218564&map_zoom=1&tree_group_layers_floor_slider=floor_slider

**Comment by arnaud-morvan** [comment]
Sorry, in fact it does not works on server side for the moment, waiting for an openshift demo 2.4 update after merge of new layer.

**Comment by sbrunner** [comment]
Cool, cela marche nikel :-)
https://camptocamp.github.io/ngeo/floorselector/examples/contribs/gmf/apps/desktop_alt.html?dim_FLOOR=8&lang=fr&tree_groups=OSM%20functions%20mixed%2CLayers%2CGroup%2COSM%20functions%2CExternal%2CFilters%20mixed%2CFilters%2CFloor%20slider&tree_group_opacity_Group=0.65&tree_enable_osm_scale=false&tree_group_layers_Layers=&tree_group_layers_Group=&tree_group_layers_OSM%20functions=&tree_enable_ch.swisstopo.dreiecksvermaschung=false&tree_enable_ch.swisstopo.geologie-gravimetrischer_atlas=false&tree_enable_ch.swisstopo.geologie-geotechnik-gk500-lithologie_hauptgruppen=false&tree_enable_ch.swisstopo.geologie-geotechnik-gk500-gesteinsklassierung=false&tree_enable_Batiment%20VD%20noWFS=false&tree_enable_ch.bfe.solarenergie-eignung-fassaden=false&tree_enable_ch.bfe.solarenergie-eignung-daecher=false&tree_enable_osm_open=false&tree_enable_OSM%20map=false&tree_enable_osm_time_r_dp_2=false&tree_group_layers_Filters=&tree_group_layers_Floor%20slider=floor_slider&baselayer_ref=OSM%20map&baselayer_opacity=0&tree_enable_Layer%20with%20very%20very%20very%20very%20very%20long%20name=false&tree_enable_osm_time_r_s=false&tree_enable_osm_time_v_s=false&tree_enable_osm_time_v_dp=false&tree_enable_osm_time_r_dp=false&tree_enable_osm_time_r_year_mounth=false&tree_enable_osm_time_d_year_mounth=false&tree_enable_osm_time_r_mounth_year=false&tree_enable_osm_time_v_mounth_year=false&tree_enable_long%20wms-t%20layer%20name%20name%20name%20name%20name%20name%20name=false&tree_enable_osm_time_r_dp_default=false&tree_enable_osm_time_r_s_default=false&tree_enable_osm_time_v_dp_default=false&tree_enable_osm_time_v_s_default=false&tree_enable_two_layers=false&tree_enable_ch.astra.ausnahmetransportrouten=false&tree_enable_no_wfs=false&map_x=573552&map_y=137885&map_zoom=2

File: contribs/gmf/src/floor/floorselectorComponent.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    }
+  }
+}
+
+
+module.component('gmfFloorselector', {
+  bindings: {
+    floor: '=',
+    items: '<',
+    size: '<'
```
Comment (fredj) [inline contribs/gmf/src/floor/floorselectorComponent.js]: Where is this variable used?

Comment (arnaud-morvan) [reply contribs/gmf/src/floor/floorselectorComponent.js]: Good catch, it was the number of visible items in first version, now size is given in css.
I will remove it.

**Comment by arnaud-morvan** [comment]
I've added the comments but I do not see the component in api doc. Do you want to investigate, or should we merge ?

**Comment by arnaud-morvan** [comment]
I got it in api doc locally, should appears in minutes on gh pages, just missing an approve now.

**Comment by arnaud-morvan** [comment]
@sbrunner @fredj : if no more comments, just missing an approve to merge this PR.

**Pull request merged**